tests/wdbcheck/datastripeoff/tests.csv

```diff
@@ -0,0 +1,2 @@
+action,ms,bytes,lang,code,repeate,minver,comment
+true,0,0,q,any@[hopen;`::36005:admin:admin;0],1,,"wdb should start up when datastriping is off"
```

Comment:
I don't think there should be hardcoded ports & user/pass in the unit tests, that makes these pretty config-specific.

Might be best to use a system command to check if revelant proc is running

(same applies to other test case)